docs/docs/technical/vaults.md

```diff
@@ -1,192 +1,186 @@
-  * **liquidation_ratio** - the percentage of user's collateral that can be liquidated at once
-  * **liquidation_penalty_liquidator** - the percentage of additional collateral going to a liquidator
-  * **liquidation_penalty_exchange** - the percentage of liquidation that goes to liquidation fund as a penalty
-  * **accumulated_interest** - interest rate of minted tokens, can be withdrawn by admin
-  * **accumulated_interest_rate** - compounded interest rate, can be used instead of compounding amount by the interest rate for every user
-  * **collateral_reserve** - address of the account to which tokens are deposited (different than reserve for [deposit](/docs/technical/collateral#deposit) to staking)
-  * **mint_amount** - the amount already minted (both amount borrowed and interest)
-  * **collateral_amount** - the amount of deposited collateral in reserve
-  * **max_borrow** - limit of total synthetic that can be borrowed
-  * **last_update** - timestamp since the last update of interest rate
-  * **bump** - used to generate the address of an account
+Data describing a vault is stored inside a _Vault_ struct. Its address is generated from addresses of tokens it uses. It is structured as following:
+
+    struct Vault {
+        pub halted: bool,
+        pub synthetic: Pubkey,
+        pub collateral: Pubkey,
+        pub debt_interest_rate: Decimal,
+        pub collateral_ratio: Decimal,
+        pub liquidation_threshold: Decimal,
+        pub liquidation_ratio: Decimal,
+        pub liquidation_penalty_liquidator: Decimal,
+        pub liquidation_penalty_exchange: Decimal,
+        pub accumulated_interest: Decimal,
+        pub accumulated_interest_rate: Decimal,
+        pub collateral_reserve: Pubkey,
+        pub mint_amount: Decimal,
+        pub collateral_amount: Decimal,
+        pub max_borrow: Decimal,
+        pub last_update: i64,
+        pub bump: u8,
+    }
 
+- **halted** - vault can be halted independently of rest of exchange (but halt of exchange affects it too)
+- **synthetic** - address of the synthetic token
+- **collateral** - address of the token used as collateral
+- **debt_interest_rate** - yearly interest rate (charged minutely)
+- **collateral_ratio** - ratio of collateral to synthetic token that can be [borrowed](#borrow) using it
+- **liquidation_threshold** - ratio of debt to the value of collateral defining when an account can be [liquidated](#liquidation)
+- **liquidation_ratio** - percentage of user's collateral that can be liquidated at once
+- **liquidation_penalty_liquidator** - percentage of additional collateral going to a liquidator
+- **liquidation_penalty_exchange** - percentage of liquidation that goes to liquidation fund as a penalty
+- **accumulated_interest** - interest rate of minted tokens. Can be withdrawn by admin
+- **accumulated_interest_rate** - compounded interest rate. Can be used instead of compounding amount by the interest rate for every user
```

Comment:
accumulated_interest_rate - it doesn't look good. It's definitely not compound interest.